Bottom: 6b4bbbd3a49f6ad909affc881d93e475234a8e90
Top:    357f3c8a1fb7fd069a2b2b0fa693a6bed406fb01
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:01:03 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
